docs/messaging/nats-integration.md

```diff
@@ -70,7 +70,31 @@ builder.AddProject<Projects.ExampleProject>()
 // After adding all resources, run the app...
 ```
 
-The NATS JetStream functionality provides a built-in persistence engine called JetStream which enables messages to be stored and replayed at a later time. You can optionally provide a `srcMountPath` parameter to specify the path to the JetStream data directory on the host machine (the provided mount path maps to the container's `-sd` argument).
+The NATS JetStream functionality provides a built-in persistence engine called JetStream which enables messages to be stored and replayed at a later time.
+
+### Add NETS server resource with authentication parameters
+
+When you want to explicitly provide the username and password, you can provide those as parameters. Consider the following alternative example:
+
+```csharp
+var builder = DistributedApplication.CreateBuilder(args);
+
+var username = builder.AddParameter("username", secret: true);
```

Comment:
```suggestion
var username = builder.AddParameter("username");
```

Typically we don't mark usernames as secret...?

Reply:
These commits mistakenly came over from a rebase, I'll incorporate the feedback elsewhere. Thank you.